<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>370</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>745</width>
    <height>370</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>745</width>
    <height>370</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DMSDataSystem</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>741</width>
      <height>311</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetMinAndMaxSize</enum>
     </property>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Record of costs</string>
        </attribute>
        <widget class="QTabWidget" name="tabWidget_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>-30</y>
           <width>739</width>
           <height>309</height>
          </rect>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>费用记录</string>
          </attribute>
          <widget class="QLabel" name="label_year">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>30</y>
             <width>55</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Year</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
          <widget class="QComboBox" name="comboBox_day">
           <property name="geometry">
            <rect>
             <x>140</x>
             <y>60</y>
             <width>55</width>
             <height>20</height>
            </rect>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>请选择</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>6</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>7</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>8</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>9</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>10</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>11</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>12</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>13</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>14</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>15</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>16</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>17</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>18</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>19</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>20</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>21</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>22</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>23</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>24</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>25</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>26</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>27</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>28</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>29</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>30</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>31</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ALL</string>
            </property>
           </item>
          </widget>
          <widget class="QLabel" name="label_day">
           <property name="geometry">
            <rect>
             <x>140</x>
             <y>30</y>
             <width>55</width>
             <height>20</height>
            </rect>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Day</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
          <widget class="QComboBox" name="comboBox_month">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>60</y>
             <width>55</width>
             <height>20</height>
            </rect>
           </property>
           <property name="editable">
            <bool>false</bool>
           </property>
           <property name="currentText">
            <string comment="                    ">请选择</string>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>请选择</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>6</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>7</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>8</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>9</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>10</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>11</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>12</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ALL</string>
            </property>
           </item>
          </widget>
          <widget class="QLineEdit" name="lineEdit_year">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>60</y>
             <width>55</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="label_month">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>30</y>
             <width>55</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Month</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
          <widget class="QTableWidget" name="table_cost">
           <property name="geometry">
            <rect>
             <x>220</x>
             <y>0</y>
             <width>511</width>
             <height>271</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="label_setCharge">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>130</y>
             <width>91</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>Set  charge:</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_charge">
           <property name="geometry">
            <rect>
             <x>95</x>
             <y>130</y>
             <width>61</width>
             <height>21</height>
            </rect>
           </property>
           <property name="inputMask">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>0.001</string>
           </property>
          </widget>
          <widget class="QWidget" name="horizontalLayoutWidget">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>200</y>
             <width>191</width>
             <height>31</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="pushButton_calculate">
              <property name="text">
               <string>Calculate</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Preferred</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_clear">
              <property name="text">
               <string>Clear</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QLabel" name="label_chargeUnit">
           <property name="geometry">
            <rect>
             <x>160</x>
             <y>130</y>
             <width>45</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>RMB / s</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>150</y>
             <width>111</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>if charge = 0  </string>
           </property>
          </widget>
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>170</y>
             <width>191</width>
             <height>18</height>
            </rect>
           </property>
           <property name="text">
            <string>charge will be set as 0.001 RMB/s</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="tab3_2">
          <attribute name="title">
           <string> 计费标准</string>
          </attribute>
         </widget>
        </widget>
       </widget>
       <widget class="QWidget" name="tab3">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <attribute name="title">
         <string>Search record</string>
        </attribute>
        <widget class="QLabel" name="label_name">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>100</y>
           <width>55</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Name</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_name">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>100</y>
           <width>115</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QWidget" name="horizontalLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>200</y>
           <width>191</width>
           <height>31</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="pushButton_search">
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_clear_2">
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QTableWidget" name="table_cost_2">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>0</y>
           <width>511</width>
           <height>271</height>
          </rect>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_times">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>20</y>
           <width>55</width>
           <height>20</height>
          </rect>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>=</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>&lt;</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>&lt;=</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>&gt;</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>&gt;=</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>&lt;&gt;</string>
          </property>
         </item>
        </widget>
        <widget class="QLabel" name="label_Times">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>54</width>
           <height>18</height>
          </rect>
         </property>
         <property name="text">
          <string>Times</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_conTimes">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>20</y>
           <width>55</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_Charge">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>50</y>
           <width>54</width>
           <height>18</height>
          </rect>
         </property>
         <property name="text">
          <string>Charge</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_charge">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>50</y>
           <width>55</width>
           <height>20</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>=</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>&lt;</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>&lt;=</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>&gt;</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>&gt;=</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>&lt;&gt;</string>
          </property>
         </item>
        </widget>
        <widget class="QLineEdit" name="lineEdit_conCharge">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>50</y>
           <width>55</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>745</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOperation">
    <property name="title">
     <string>Operation</string>
    </property>
    <addaction name="actionChange_skin"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuOperation"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionChange_skin">
   <property name="icon">
    <iconset>
     <normaloff>:/changeskin</normaloff>:/changeskin</iconset>
   </property>
   <property name="text">
    <string>Change skin</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset>
     <normaloff>:/quit</normaloff>:/quit</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset>
     <normaloff>:/about</normaloff>:/about</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit_charge</tabstop>
  <tabstop>pushButton_calculate</tabstop>
  <tabstop>comboBox_month</tabstop>
  <tabstop>pushButton_clear</tabstop>
  <tabstop>comboBox_day</tabstop>
  <tabstop>lineEdit_year</tabstop>
  <tabstop>table_cost</tabstop>
  <tabstop>tabWidget_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
